tests/test_boolean.py

```diff
@@ -324,217 +195,419 @@ def test_parse_side_by_side_symbols_with_parens_raise_exception(self):
+            BaseElement('a')
+
+    def test_true_and_false(self):
         algebra = BooleanAlgebra()
-        self.assertEqual(algebra.TRUE, algebra.TRUE)
-        BaseElement()
-        self.assertRaises(TypeError, BaseElement, 2)
-        self.assertRaises(TypeError, BaseElement, 'a')
-        self.assertTrue(algebra.TRUE is algebra.TRUE)
-        self.assertTrue(algebra.TRUE is not algebra.FALSE)
-        self.assertTrue(algebra.FALSE is algebra.FALSE)
-        self.assertTrue(bool(algebra.TRUE) is True)
-        self.assertTrue(bool(algebra.FALSE) is False)
-        self.assertEqual(algebra.TRUE, True)
-        self.assertEqual(algebra.FALSE, False)
+
+        assert algebra.TRUE is algebra.TRUE
+        assert algebra.FALSE is algebra.FALSE
+
+        assert algebra.TRUE == algebra.TRUE
+        assert algebra.FALSE == algebra.FALSE
+
+        assert algebra.TRUE is not algebra.FALSE
+        assert algebra.FALSE is not algebra.TRUE
+
+        assert algebra.TRUE != algebra.FALSE
+        assert algebra.FALSE != algebra.TRUE
+
+        assert bool(algebra.TRUE) is True
+        assert bool(algebra.FALSE) is False
+
+        assert algebra.TRUE == True
+        assert algebra.FALSE == False
 
     def test_literals(self):
         algebra = BooleanAlgebra()
-        self.assertEqual(algebra.TRUE.literals, set())
-        self.assertEqual(algebra.FALSE.literals, set())
+
+        assert algebra.TRUE.literals == set()
+        assert algebra.FALSE.literals == set()
 
     def test_literalize(self):
         algebra = BooleanAlgebra()
-        self.assertEqual(algebra.TRUE.literalize(), algebra.TRUE)
-        self.assertEqual(algebra.FALSE.literalize(), algebra.FALSE)
+
+        assert algebra.TRUE.literalize() == algebra.TRUE
+        assert algebra.FALSE.literalize() == algebra.FALSE
+
+        assert algebra.TRUE.literalize() != algebra.FALSE
+        assert algebra.FALSE.literalize() != algebra.TRUE
 
     def test_simplify(self):
         algebra = BooleanAlgebra()
-        self.assertEqual(algebra.TRUE.simplify(), algebra.TRUE)
-        self.assertEqual(algebra.FALSE.simplify(), algebra.FALSE)
+
+        assert algebra.TRUE.simplify() == algebra.TRUE
+        assert algebra.FALSE.simplify() == algebra.FALSE
 
     def test_dual(self):
         algebra = BooleanAlgebra()
-        self.assertEqual(algebra.TRUE.dual, algebra.FALSE)
-        self.assertEqual(algebra.FALSE.dual, algebra.TRUE)
+
+        assert algebra.TRUE.dual ==  algebra.FALSE
+        assert algebra.FALSE.dual ==  algebra.TRUE
 
     def test_equality(self):
         algebra = BooleanAlgebra()
-        self.assertEqual(algebra.TRUE, algebra.TRUE)
-        self.assertEqual(algebra.FALSE, algebra.FALSE)
-        self.assertNotEqual(algebra.TRUE, algebra.FALSE)
+
+        assert algebra.TRUE == algebra.TRUE
+        assert algebra.FALSE == algebra.FALSE
+
+        assert     algebra.TRUE != algebra.FALSE
+        assert not algebra.TRUE == algebra.FALSE
```

Comment:
Was writing like this first:
```
assert not (algebra.TRUE == algebra.FALSE)
```

and then noticed that the `()` can be dropped, so I dropped them. Will rewrite as:

```
        assert      algebra.TRUE != algebra.FALSE
        assert not (algebra.TRUE == algebra.FALSE)
```